app/services/irs_attempts_api/attempt_event.rb

```diff
@@ -41,8 +41,8 @@ def self.from_jwe(jwe, private_key)
         iat: parsed_event['iat'],
         event_type: event_type,
         session_id: event_data['subject']['session_id'],
-        occurred_at: Time.zone.at(event_data['occurred_at']),
-        event_metadata: event_data.symbolize_keys.except(:subject, :occurred_at),
+        occurred_at_ms: Time.zone.at(event_data['occurred_at_ms']),
```

Comment:
ditto that I think they want ms since the epoch, so we need to divide here:
```suggestion
        occurred_at_ms: Time.zone.at(event_data['occurred_at_ms'] / 1000.0),
```

Reply:
I'm not sure I follow. I interpreted their request as just adding fractional (decimal) seconds, e.g.:

```
[1] pry(main)> Time.zone.now.to_f
=> 1668637667.910866
```

Reply:
following back up: we chatted and are going to stick with fractional seconds, so keeping the `.to_f` no multiplying by 1000